### PR TITLE
Update cloud-data-export.rst

### PR DESCRIPTION
**Pull request by justinegeffen**
Update for: https://mattermost.atlassian.net/browse/MM-41264

### CODE REVIEW

File: source/manage/cloud-data-export.rst [hunk 1]
```diff
@@ -25,6 +25,11 @@ You'll be using the `mmctl export <https://docs.mattermost.com/manage/mmctl-comm
 
 Once mmctl is installed, we can generate the export from the source instance.
 
+AWS Region
+----------
+
+Currently, Mattermost Cloud data resides in the aws-us-east-1 region. Please contact feedback-cloud[at]mattermost.com to share feedback about additional data residencies with the product team.
```
Comment (jasonblais) [inline source/manage/cloud-data-export.rst:31]:
```suggestion
Currently, Mattermost Cloud data resides in the `aws-us-east-1` region, located in Virginia, United States. Please contact the product team via feedback-cloud[at]mattermost.com if you have additional data residency requirements.
```

**Review by jasonblais — CHANGES_REQUESTED** [review]
Thanks @justinegeffen! Given this page is designs for Self-Hosted admins who are looking to migrate to Cloud, I wonder if there is another location we could add this information to, which caters to any Cloud users? (e.g. those coming from Slack)

**Comment by justinegeffen** [comment]
> Thanks @justinegeffen! Given this page is designs for Self-Hosted admins who are looking to migrate to Cloud, I wonder if there is another location we could add this information to, which caters to any Cloud users? (e.g. those coming from Slack)

We could also add it to: https://docs.mattermost.com/onboard/migrating-to-mattermost.html as a general note.

**Comment by jasonblais** [comment]
(I'm happy to submit a PR for it for review)

**Comment by justinegeffen** [comment]
Ah, gotcha. Sorry for the misunderstanding there! That draft looks good to me; let me find the right place for it to live.

**Pull request closed without merging**